airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/task/internal/InputConsumerTask.kt

```diff
@@ -83,7 +83,8 @@ class InputConsumerTask(
         log.info { "Starting consuming messages from the input flow" }
         val unopenedStreams = catalog.streams.map { it.descriptor }.toMutableSet()
         pipelineInputQueue.use {
-            pipelineEventBookkeepingRouter.use {
+            var baseException: Exception? = null
```

Comment:
nit: maybe rename this `pipelineException` or `processingException`, etc to be a little more descriptive